docs-metadata/src/controllers/apply-controller.ts

```diff
@@ -0,0 +1,52 @@
+
+import {commands, 
+	env, 
+	window, 
+	workspace, 
+	ExtensionContext, 
+	OpenDialogOptions,
+	Uri} from 'vscode';
+
+export async function showApplyMetadataMessage(folderPath:string)
+{
+	window.showInformationMessage(`Apply MUT file ${folderPath}?`, 
```

Comment:
we should probably spell out the entire word instead of using MUT

Reply:
Hmmm... didn't think about it before by I agree. This message was in the spec specifically.